css/properties/paint-order.json

```diff
@@ -0,0 +1,68 @@
+{
+  "css": {
+    "properties": {
+      "paint-order": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/paint-order",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
```

Comment:
@chrisdavidmills It looks like we do support paint-order. https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/css/CSSProperties.json5?l=2067

Reply:
Hrm. Is it behind a pref/flag? I've tried the example at https://developer.mozilla.org/en-US/docs/Web/CSS/paint-order in the latest Canary, and didn't get any joy.

Reply:
Leave it false then. 

Our meta documents are usually pretty reliable indicators of functionality. This is not apparently. I'll dig into this and submit my own PR if need be.